bindings/generate_bindings.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import subprocess
+from pathlib import Path
+
+def uniffi_gen(executable: str, *args):
+    args = list(str(a) for a in args)
+    uniffi = [executable]
+    uniffi.extend(args)
+    print(f'Running {" ".join(uniffi)}')
+    subprocess.run(uniffi, check=True)
+
+def uniffi_gen_go(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen-go", udl, "-o", output_dir)
+
+
+def uniffi_gen_cs(udl: Path, output_dir: Path, config_file: Path):
+    uniffi_gen("uniffi-bindgen-cs", udl, "-o", output_dir, "-c", config_file)
+
+
+def uniffi_gen_swift(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen", "generate", udl, "--language", "swift", "-o", output_dir)
+
+
+def uniffi_gen_kotlin(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen", "generate", udl, "--language", "kotlin", "-o", output_dir)
+
+def uniffi_gen_python(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen", "generate", udl, "--language", "python", "-o", output_dir)
+
+def uniffi_gen_cpp(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen-cpp", udl, "-o", output_dir)
```

Comment:
do we have someone who uses the c++ bindings?

Reply:
No, but we agreed to keep them for now. The same situation with python ones.

Reply:
I don't think so. But keeping them is no harm either